docs/GettingStarted/configuration/PelorusCore.md

```diff
@@ -0,0 +1,222 @@
+# Configuration
+
+The Pelorus Core configuration applies to Prometheus, Grafana, Thanos and other operational aspects of the Pelorus stack.
+
+Those configuration options are used in the Pelorus configuration object YAML file to create Pelorus application instance.
+
+Each Pelorus Core configuration option must be placed under `spec` in the Pelorus configuration object YAML file as in the example:
+
+```yaml
+kind: Pelorus
+apiVersion: charts.pelorus.konveyor.io/v1alpha1
+metadata:
+  name: pelorus-pelorus-instance
+  namespace: pelorus
+spec:
+  exporters:
+    [...] # Pelorus exporters configuration options
+  # Pelorus Core configuration options
+```
+
+For configuration options for exporters, check [its configuration guide](PelorusExporters.md).
+
+## Example
+
+Configuration part of the Pelorus object YAML file, with some non-default options:
+
+```yaml
+kind: Pelorus
+apiVersion: charts.pelorus.konveyor.io/v1alpha1
+metadata:
+  name: pelorus-instance
+  namespace: pelorus
+spec:
+  exporters:
+    [...] # Pelorus exporters configuration options
+  # Below are non-default options
```

Comment:
I would remove this comment, this information is already in the previous paragraph